app/src/main/java/org/sopt/cgv/feature/seats/component/Stepper.kt

```diff
@@ -0,0 +1,108 @@
+package org.sopt.cgv.feature.seats.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.cgv.R
+import org.sopt.cgv.core.common.extension.noRippleClickable
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.Gray100
+import org.sopt.cgv.core.designsystem.theme.Gray700
+import org.sopt.cgv.core.designsystem.theme.Gray800
+import org.sopt.cgv.core.designsystem.theme.White
+
+@Composable
+fun Stepper(
+    modifier: Modifier = Modifier,
+    label: String,
+    viewModel: SeatSelectViewModel
```

Comment:
```suggestion
    value: Int,                
    onIncrease: () -> Unit,    
    onDecrease: () -> Unit     
```
여기서도 지금까지와 같은 이유로 이렇게 바꾸면 좋을 거 같아요! 
추가적으로 label이 ViewModel의 stepperValues Map에서 해당 Stepper의 값을 찾는 키로 사용되고 있는 거 같은데 맞을까요? 위와 같이 바꾸면 value를 직접 받아오므로 Map에서 값을 찾을 필요가 없어져서 아래 코드가 필요 없어지게 될 거 같은데 한번 확인해주시면 감사하겠습니다!